tests/app/helpers/test_template_helpers.py

```diff
@@ -149,6 +178,16 @@ def test_footer_warning_not_in_context_census_theme(app: Flask):
                 "title": "Test",
             },
         ),
+        (
+            SurveyType.SOCIAL,
+            "Test",
+            SocialSurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "Test",
+            },
+        ),
```

Comment:
Is this not identical to the one above?. Need to test the Welsh version.

Reply:
Ok, testing the Welsh, added cy logo and test. Cheers!

Reply:
I don't see anything around the welsh logo change.

Reply:
Yeah I had to remove `page_header_logo` after Rich merged his into master. I will check Welsh branding.

Reply:
So I spoke to Rich, the welsh logo is set automatically based on the current language code in the language object that gets added for the language switch link, so after removing `page_header_logo` no further changes needed.